home-manager/bash.nix

```diff
@@ -77,12 +77,25 @@
 
     # Switch to another shell when bash used as a login shell
     profileExtra = ''
+      # Don't extract an alias or command. Write in each shells
+      # Read package.nix why using different ps command in Linux and Darwin
+      parent_command=""
+      case ''${OSTYPE} in
+      linux*)
+        parent_command="$(${pkgs.procps}/bin/ps --no-header --pid=$PPID --format=comm)"
+        ;;
+      darwin*)
```

Comment:
Now zsh is the default in darwin, before of https://github.com/kachick/dotfiles/pull/509 bash error log made confusion for me...